dashboard/app/mailers/pd/workshop_mailer.rb

```diff
@@ -87,22 +87,34 @@ def teacher_enrollment_reminder(enrollment)
       reply_to: email_address(@workshop.organizer.name, @workshop.organizer.email)
   end
 
-  # TODO: Make sure every teacher has an enrollment
   def exit_survey(workshop, teacher, enrollment)
     @workshop = workshop
     @teacher = teacher
     @is_first_workshop = Pd::Workshop.attended_by(teacher).in_state(Pd::Workshop::STATE_ENDED).count == 1
 
-    @survey_url = CDO.code_org_url "/pd-workshop-survey/#{enrollment.code}"
+    @survey_url = CDO.code_org_url "/pd-workshop-survey/#{enrollment.code}", 'https:'
```

Comment:
Yes, in the [exit_survey](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/views/pd/workshop_mailer/exit_survey.html.haml#L25) mail view.